src/ms.tex

```diff
@@ -1241,27 +1262,31 @@ \subsection{Current and Future Challenges}
 \astropy, who have opened issues and provided feedback, and have supported the
 project in a number of different ways.
 
+We acknowledge the Gordon and Betty Moore foundation for their continued
+financial support.
+This work is partially supported by NASA under Grant No. 20-OSTFL20-0057 issued
+through the NASA ROSES program.
```

Comment:
@eteq Can you audit this grant number? Thanks!

Reply:
Never mind - I found it here: https://www.nasa.gov/centers/nssc/forms/grant-status-form (searching for Tollerud)